daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -66,3 +66,4 @@ This table is meant to help users understand the equivalent options for running
 | not supported | not supported | | `dapr.io/volume-mounts` | List of pod volumes to be mounted to the sidecar container in read-only mode. Strings consisting of `volume:path` pairs separated by a comma. Example, `"volume-1:/tmp/mount1,volume-2:/home/root/mount2"`. |
 | not supported | not supported | | `dapr.io/volume-mounts-rw` | List of pod volumes to be mounted to the sidecar container in read-write mode. Strings consisting of `volume:path` pairs separated by a comma. Example, `"volume-1:/tmp/mount1,volume-2:/home/root/mount2"`. |
 | `--disable-builtin-k8s-secret-store` | not supported | | `dapr.io/disable-builtin-k8s-secret-store` | Disables BuiltIn Kubernetes secret store. Default value is false. See [Kubernetes secret store component]({{<ref "kubernetes-secret-store.md">}}) for details. |
+| not supported | not supported | | `dapr.io/sidecar-seccomp-profile-type` | Set sidecar container's `securityContext.seccompProfile.type` to `Unconfined`, `RuntimeDefault`, or `Localhost`. By default, the field will be ommited from sidecar container. |
```

Comment:
For this sentence, 

"By default, the field will be omited from sidecar container."
Don't you mean to say
"By default, this annotation is not set on the Dapr sidecar."